Gemfile.lock

```diff
@@ -54,6 +55,10 @@ GEM
       unicode-display_width (~> 1.0, >= 1.0.1)
     ruby-progressbar (1.8.1)
     sass (3.4.22)
```

Comment:
libsass 3.3 still targets compatibility with sass 3.4. It appears to be reasonably close. I don't think we document any particular version of the sass spec: https://developer.zendesk.com/apps/docs/agent/styling